features/step_definitions/git-flow.js

```diff
@@ -143,16 +159,77 @@ defineSupportCode(({ Before, Given, When, Then }) => {
             });
     });
 
-    Then(/^a new build from `main` should be created to test that change$/, {
+    When(/^a tag is created$/, {
         timeout: TIMEOUT
     }, function step() {
+        const branch = this.branch;
+        const tag = this.tag;
+
+        return github.createBranch(branch, this.repoOrg, this.repoName)
+            .then(() => github.createFile(branch, this.repoOrg, this.repoName))
+            .then(({ data }) => {
+                this.sha = data.commit.sha;
+                github.createTag(tag, branch, this.repoOrg, this.repoName);
+            });
+    });
+
+    When(/^a annotated tag is created$/, {
+        timeout: TIMEOUT
+    }, function step() {
+        const branch = this.branch;
+        const tag = this.tag;
+
+        return github.createBranch(branch, this.repoOrg, this.repoName)
+            .then(() => github.createFile(branch, this.repoOrg, this.repoName))
+            .then(({ data }) => {
+                this.sha = data.commit.sha;
+                github.createAnnotatedTag(tag, branch, this.repoOrg, this.repoName);
+            });
+    });
+
+    When(/^a release is created$/, {
+        timeout: TIMEOUT
+    }, function step() {
+        const branch = this.branch;
+        const tag = this.tag;
+
+        return github.createBranch(branch, this.repoOrg, this.repoName)
+            .then(() => github.createFile(branch, this.repoOrg, this.repoName))
+            .then(({ data }) => {
+                this.sha = data.commit.sha;
+
+                return github.createTag(tag, branch, this.repoOrg, this.repoName);
+            })
+            .then(() => github.createRelease(tag, this.repoOrg, this.repoName));
+    });
+
+    When(/^a annotated release is created$/, {
+        timeout: TIMEOUT
+    }, function step() {
+        const branch = this.branch;
+        const tag = this.tag;
+
+        return github.createBranch(branch, this.repoOrg, this.repoName)
+            .then(() => github.createFile(branch, this.repoOrg, this.repoName))
+            .then(({ data }) => {
+                this.sha = data.commit.sha;
+
+                return github.createAnnotatedTag(tag, branch, this.repoOrg, this.repoName);
+            })
+            .then(() => github.createRelease(tag, this.repoOrg, this.repoName));
+    });
+
+    Then(/^a new build from "([^"]*)" should be created to test that change$/, {
```

Comment:
This Regular expression is matched with `""`.
How about below?
`"([^"]*)"` => `"([^"]+)"`

Reply:
Thanks. I fixed it.